src/javascripts/menu items/menu.js

```diff
@@ -16,4 +16,24 @@ const showMenuItems = (array) => {
   });
 };
 
-export default showMenuItems;
+const showUserMenuItems = (array) => {
```

Comment:
nit: The naming of this function is confusing. It reads as if the function returns menu items that belong to a user.